### PR TITLE
Ticket import from csv file

### PR DESCRIPTION
**Pull request by ercasta**
Allows importing an external csv with tickets for an event. Participants are automatically created if not already in the DB

### CODE REVIEW

File: templates/coderdojomobile/eventTicketUpload.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    
+    <p class="text-center" >{{event.description}}</p>
+
+    <ul>
+        {% for ticket in tickets %}
+        <li><span style="color:green">{% bootstrap_icon "ok-circle" %}</span>
+        {% blocktrans with name=ticket.participant.name|title surname=ticket.participant.surname|title uuid=ticket.uuid|title %}Ticket_created {{name}} {{surname}} {{uuid}} {% endblocktrans %}
+        {% endfor %}
+    </ul>
+    <a href="{% url 'coderdojomobile:eventDetails' event.id %}"/>{% trans "Back to the event"%}</a>
```
Comment (sandrobonazzola) [inline templates/coderdojomobile/eventTicketUpload.html]: here I see another terminated <a/>

File: templates/coderdojomobile/eventTicketUpload.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{% else %}
+    <div class="row">
+        <div class="col-md-12">{% trans "Event_not_found" %}</div>
+    </div>
+{% endif %}
+
+</div>
+
+
+{% endblock content %}
```
Comment (sandrobonazzola) [inline templates/coderdojomobile/eventTicketUpload.html]: Line not terminated

File: views.py [hunk 3]
```diff
@@ -450,6 +457,118 @@ def eventCheckInOut(request, event_id):
...
+        '''
+        CSV format support 4 fields:
+        name, surname, ticket id (optional), badge id (optional)
+        '''
+        ticket_list = []
+        if form.is_valid():
+            try:
+                event = Event.objects.get(id=event_id)
+            except Event.DoesNotExist:
+                None
```
Comment (sandrobonazzola) [inline views.py]: Please use "pass" if nothing to be done. I would have preferred to see some error on the screen.

File: views.py [hunk 4]
```diff
@@ -450,6 +457,118 @@ def eventCheckInOut(request, event_id):
...
+    surname = ticket['Surname']
+    ticket_id = ticket['Ticket_id']
+    badge_id = ticket['Badge_id']
+    participant = None
+    if badge_id is not None and len(badge_id) > 0:
+        # Look for the Participant
+        try:
+            participant = Participant.objects.get(uuid=badge_id)
+        except Participant.DoesNotExist:
+            None
```
Comment (sandrobonazzola) [inline views.py]: Please use "pass" and not None if nothing has to be done. I would prefer an error message.

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]

**Comment by ercasta** [comment]
Implemented requested changes

**Review by sandrobonazzola — APPROVED** [review]
Thanks!

**Pull request merged**